distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Java_Problems.enso

```diff
@@ -26,6 +26,7 @@ polyglot java import org.enso.table.parsing.problems.AdditionalInvalidRows
 polyglot java import org.enso.table.parsing.problems.InvalidFormat
 polyglot java import org.enso.table.parsing.problems.InvalidRow
 polyglot java import org.enso.table.problems.ProblemAggregator
+polyglot java import org.enso.table.problems.ProblemAggregator.ProblemSummary
```

Comment:
There is a call to `ProblemSummary` instance method in this file. Importing to make the class ready for reflection access.